lib/ecmascript.ts

```diff
@@ -5085,11 +5133,31 @@ export function CreateOnePropObject<K extends string, V>(propName: K, propValue:
 }
 
 function GetOption<P extends string, O extends Partial<Record<P, unknown>>>(
```

Comment:
I'm not familiar enough with TypeScript to know what these declarations do if they don't have function bodies. Do they just serve as alternate type signatures for the function below that does have a body?